tests/eval/bad/gold/builtin-ecdsa_recover.scilexp.gold

```diff
@@ -0,0 +1,11 @@
+{
+  "gas_remaining": "4001696",
+  "errors": [
+    {
+      "error_message": "Sign.read_recoverable_exn: recid must be 0, 1, 2 or 3",
+      "start_location": { "file": "", "line": 0, "column": 0 },
```

Comment:
nit: Is it possible to set a location for such messages in the evaluator?

Reply:
Good point! Let's address this in a separate PR targeting #1135